client/blocks/login/social.jsx

```diff
@@ -182,17 +194,25 @@ class SocialLoginForm extends Component {
 			<p className="login__social-tos">
 				{ translate(
 					"If you continue with Google or Apple and don't already have a WordPress.com account, you" +
-						' are creating an account and you agree to our' +
-						' {{a}}Terms of Service{{/a}}.',
+						' are creating an account, you agree to our' +
+						' {{tosLink}}Terms of Service{{/tosLink}}, and acknowledge that you have' +
+						' read our {{privacyLink}}Privacy Policy{{/privacyLink}}.',
 					{
 						components: {
-							a: (
+							tosLink: (
 								<a
 									href={ localizeUrl( 'https://wordpress.com/tos/' ) }
 									target="_blank"
 									rel="noopener noreferrer"
 								/>
 							),
+							privacyLink: (
```

Comment:
I realize some of that repetition was here before, but it's a bummer to repeat those a few times in the same component. Could we try to reuse them?

Reply:
Feel free to follow up with that in another PR if that makes it easier FWIW.

Reply:
Thanks! It's a good suggestion 👍 Changed in https://github.com/Automattic/wp-calypso/pull/66220/commits/b0bb2d9b6e39bf15981db9cdc201db9417fd07ce